### ISSUE TITLE
Inability to handle env var references (e.g. a command that uses $SOME_VAR

### ISSUE THREAD

**Issue by phillipedwards** [pulumi/kube2pulumi#100]
### What happened?

kube2pulumi is not able to handle env var references such as a command containing an env var

### Example

```
    command:
        - sh
        - -ec
        # The statically linked Go program binary is invoked to avoid any
        # dependency on utilities like sh and mount that can be missing on certain
        # distros installed on the underlying host. Copy the binary to the
        # same directory where we install cilium cni plugin so that exec permissions
        # are available.
        - |
          cp /usr/bin/cilium-mount /hostbin/cilium-mount;
          nsenter --cgroup=/hostproc/1/ns/cgroup --mount=/hostproc/1/ns/mnt "${BIN_PATH}/cilium-mount" $CGROUP_ROOT;
          rm /hostbin/cilium-mount
```

### Output of `pulumi about`

kube2pulumi 0.0.15

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by mikhailshilkov** [comment]
@phillipedwards Could you please specify the error that you are getting?

**Comment by rquitales** [comment]
This is fixed in #96 and in release v0.0.15.

Converting 
```yaml
        - |
          cp /usr/bin/cilium-mount /hostbin/cilium-mount;
          nsenter --cgroup=/hostproc/1/ns/cgroup --mount=/hostproc/1/ns/mnt "${BIN_PATH}/cilium-mount" $CGROUP_ROOT;
          rm /hostbin/cilium-mount
```

Results in:

```go
Command: pulumi.StringArray{
pulumi.String(fmt.Sprintf("cp /usr/bin/cilium-mount /hostbin/cilium-mount;\n        nsenter --cgroup=/hostproc/1/ns/cgroup --mount=/hostproc/1/ns/mnt \"${BIN_PATH}/cilium-mount\" $CGROUP_ROOT;\n        rm /hostbin/cilium-mount\n")),
},
```
